plugins/openai/oaievalset.go

```diff
@@ -0,0 +1,25 @@
+package openai
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func OpenAIEvalSetCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "OpenAI Eval Set CLI",
+		Runs:    []string{"oaievalset"},
+		DocsURL: sdk.URL("https://github.com/openai/evals/blob/main/README.md"),
```

Comment:
Is there a more direct link to `oaievalset`? I don't see it even mentioned there.

This one, maybe?
```suggestion
		DocsURL: sdk.URL("https://github.com/openai/evals/blob/main/docs/run-evals.md"),
```

Reply:
Thanks for the feedback, @florisvdg. Agree that the other link seems to be a better choice. I have updated both `oaieval` and `oaievalset` to it: a1e226c69fe93fd62d0836c12af81b3fa6d80530.